packages/portfolio-contract/src/type-guards-steps.ts

```diff
@@ -61,6 +61,7 @@ export type MovementDesc = {
   fee?: NatAmount;
   /** for example: { usdnOut: 98n } */
   detail?: Record<string, NatValue>;
+  claim?: boolean;
```

Comment:
The corresponding shape/pattern needs to change.

Why didn't this get caught by tests?
oh! `openPortfolio` assumes the caller has done pattern validation.

We need to test this in `offer-shapes.test.ts`